NuRadioReco/utilities/signal_processing.py

```diff
@@ -82,3 +82,43 @@ def add_cable_delay(station, det, sim_to_data=None, trigger=False, logger=None):
                         f"of cable delay to channel {channel.get_id()}")
 
         channel.add_trace_start_time(add_or_subtract * cable_delay)
+
+
+def calculate_vrms_from_temperature(temperature, bandwidth=None, response=None, resistance=50):
+    """ Helper function to calculate the noise vrms from a given noise temperature and bandwidth.
+
+    For details see https://en.wikipedia.org/wiki/Johnson%E2%80%93Nyquist_noise
+    (sec. "Maximum transfer of noise power") or our wiki
+    https://nu-radio.github.io/NuRadioMC/NuRadioMC/pages/HDF5_structure.html
+
+    Parameters
+    ----------
+    temperature: float
+        The noise temperature of the channel in Kelvin
+    bandwidth: tuple (list of 2 floats) (default: None)
```

Comment:
can we rather have this be the "effective_bandwidth"? I.e., the integral over the squared S21? 
In the most simple case it would just be f_max - f_min. 
Then this function is directly usable from simulation.py